source/jormungandr/jormungandr/interfaces/v1/Uri.py

```diff
@@ -585,15 +594,25 @@ class Disruptions(Uri):
 
         def __init__(self):
             Uri.__init__(self, is_collection, "disruptions")
-            self.collections = [
-                ("pagination", PbField(pagination)),
-                ("error", PbField(error)),
-                ("disruptions", fields.List(NonNullNested(disruption_marshaller), attribute="impacts")),
-                ("feed_publishers", NonNullList(fields.Nested(feed_publisher,
-                                           display_null=False)))
-            ]
-            collections = marshal_with(OrderedDict(self.collections),
-                                       display_null=False)
+            #self.collections = [
+            #    ("pagination", PbField(pagination)),
+            #    ("error", PbField(error)),
+            #    ("disruptions", fields.List(NonNullNested(disruption_marshaller), attribute="impacts")),
+            #    ("feed_publishers", NonNullList(fields.Nested(feed_publisher,
+            #                               display_null=False)))
+            #]
+            #collections = marshal_with(OrderedDict(self.collections),
+            #                           display_null=False)
+
+            from jormungandr.interfaces.v1.serializer import serialize_with, ErrorSerializer, DisruptionSerializer, FeedPublisherSerializer, PaginationSerializer
+            import serpy
+            class Serializer(serpy.Serializer):
```

Comment:
hmmm, how about moving `class Serializer(serpy.Serializer)` into jormungandr.interfaces.v1.serializer and renaming it to  `DisruptionsSerializer`? thus, you don't need to import all these staffs and same for the codes above